main/modes/mainMenu/mainMenu.c

```diff
@@ -219,12 +219,12 @@ static void mainMenuCb(const char* label, bool selected, uint32_t settingVal)
         else if (bgmVolSettingLabel == label)
         {
             setBgmVolumeSetting(settingVal);
-            bzrPlayBgm(&mainMenu->jingle);
+            bzrPlaySfx(&mainMenu->jingle, BZR_LEFT);
```

Comment:
Shouldn't this still be `bzrPlayBgm`?

Reply:
Yeah

Reply:
And it should be `BZR_STEREO`. I was only using L/R for testing here